modules/contentbox/models/content/BaseContent.cfc

```diff
@@ -1236,7 +1236,7 @@ component 	persistent="true"
 	* Build content cache keys according to sent content object
 	*/
 	string function buildContentCacheKey(){
-		return "cb-content-#cgi.http_host#-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#";
+		return "cb-content-#cgi.http_host#-#cgi.query_string#-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#";
```

Comment:
Hmm, I am not sure how I feel about this one, since potentially any query string can alter the translation cache.  What is your reasoning behind this?